README.md

```diff
@@ -15,108 +12,121 @@ Each statefulset uses PodAntiAffinity to ensure that the master and slaves are d
 At the same time, when the operator selects the master in each statefulset, it preferentially select the pod
 with different k8s nodes as master.
 
-Table of Contents
-=================
 
-   * [redis-cluster-operator](#redis-cluster-operator)
-      * [Overview](#overview)
-   * [Table of Contents](#table-of-contents)
-      * [Prerequisites](#prerequisites)
-      * [Features](#features)
-      * [Quick Start](#quick-start)
-         * [Deploy redis cluster operator](#deploy-redis-cluster-operator)
-            * [Install Step by step](#install-step-by-step)
-            * [Install using helm chart](#install-using-helm-chart)
-         * [Usage](#usage)
-            * [Deploy a sample Redis Cluster](#deploy-a-sample-redis-cluster)
-            * [Scaling Up the Redis Cluster](#scaling-up-the-redis-cluster)
-            * [Scaling Down the Redis Cluster](#scaling-down-the-redis-cluster)
-            * [Backup and Restore](#backup-and-restore)
-            * [Prometheus Discovery](#prometheus-discovery)
-            * [Create Redis Cluster with password](#create-redis-cluster-with-password)
-            * [Persistent Volume](#persistent-volume)
-            * [Custom Configuration](#custom-configuration)
-            * [Custom Service](#custom-service)
-            * [Custom Resource](#custom-resource)
-      * [ValidatingWebhook](#validatingwebhook)
-      * [End to end tests](#end-to-end-tests)
+## Table of Contents
+
+- [Overview](#redis-cluster-operator)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [To Deploy on the Cluster](#to-deploy-on-the-cluster)
+  - [To Uninstall](#to-uninstall)
+- [Project Distribution](#project-distribution)
+  - [Build Installer](#build-the-installer)
+  - [Using the Installer](#using-the-installer)
+- [Usage](#usage)
+  - [Deploy a Sample Redis Cluster](#deploy-a-sample-redis-cluster)
+  - [Scaling Up the Redis Cluster](#scaling-up-the-redis-cluster)
+  - [Scaling Down the Redis Cluster](#scaling-down-the-redis-cluster)
+  - [Create cleanup job](#create-cleanup-job)
+  - [Backup and Restore](#backup-and-restore)
+  - [Prometheus Discovery](#prometheus-discovery)
+  - [Create Redis Cluster with password](#create-redis-cluster-with-password)
+  - [Persistent Volume](#persistent-volume)
+  - [Custom Configuration](#custom-configuration)
+  - [Custom Service](#custom-service)
+  - [Custom Resource](#custom-resource)
+- [Contributing](#contributing)
+- [End to End Tests](#end-to-end-tests)
 
-## Prerequisites
+## Getting Started
 
-* go version v1.13+.
-* Access to a Kubernetes v1.13.10 cluster.
+### Prerequisites
+- go version v1.22.0+
+- docker version 17.03+.
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
 
-## Features
```

Comment:
I'add this featrues section so that folks will know what is available in this repo